linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/RequestFramingFilter.scala

```diff
@@ -0,0 +1,46 @@
+package io.buoyant.linkerd.protocol.http
+
+import com.twitter.finagle._
+import com.twitter.finagle.http.{Request, Response, Status}
+import com.twitter.finagle.buoyant.linkerd.Headers
+import com.twitter.logging.Logger
+import com.twitter.util.Future
+
+/**
+ * A filter that fails badly-framed requests.
+ */
+class RequestFramingFilter extends SimpleFilter[Request, Response] {
+  private[this] lazy val log = Logger.get("RequestFramingFilter")
```

Comment:
Doing upfront initialization of the Logger is probably fine and lets us avoid the evil `lazy` keyword.